### PR TITLE
chore: Mask Wallet Integration

### PR DESCRIPTION
**Pull request by guanbinrui**
```typescript
// sdk
export { importMnemonic, importPrivateKey } from './maskwallet'

// db
export { getWallet, getWallets, removeWallet, updateWallet, hasWallet } from './database/wallet'

// password
export { hasPassword, verifyPassword, changePassword, validatePassword, clearPassword } from './password'

// locker
export { isLocked, lockWallet, unlockWallet } from './locker'
```
closes #

### CODE REVIEW

File: packages/maskbook/src/extension/background-script/WalletService.ts [hunk 1]
```diff
@@ -0,0 +1 @@
+export * from './WalletService'
```
Comment (guanbinrui) [inline packages/maskbook/src/extension/background-script/WalletService.ts:1]: Remoke this.

File: packages/maskbook/src/plugins/Wallet/services/wallet/index.ts [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+})
+
+type Request = InstanceType<typeof api.MWRequest>
+type Response = InstanceType<typeof api.MWResponse>
+
+function send<I extends keyof Request, O extends keyof Response>(input: I, output: O) {
+    return (value: Request[I]) => {
+        return new Promise<Response[O]>((resolve, reject) => {
+            const payload: Uint8Array = api.MWRequest.encode({ [input]: value }).finish()
+            worker.addEventListener('message', (ev: MessageEvent) => {
```
Comment (guanbinrui) [inline packages/maskbook/src/plugins/Wallet/services/wallet/index.ts:22]: Need to remove listeners.

File: packages/maskbook/src/extension/service.ts [hunk 3]
```diff
@@ -29,6 +29,7 @@ export const Services = {
     Steganography: add(() => import('./background-script/SteganographyService'), 'Steganography'),
     Helper: add(() => import('./background-script/HelperService'), 'Helper'),
     Provider: add(() => import('./background-script/ProviderService'), 'Provider'),
+    Wallet: add(() => import('./background-script/WalletService'), 'Wallet'),
```
Comment (guanbinrui) [inline packages/maskbook/src/extension/service.ts:32]: Revoke this.

File: packages/maskbook/src/plugins/Wallet/services/wallet/index.ts [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+import { OnDemandWorker } from '../../../../web-workers/OnDemandWorker'
+
+let worker: OnDemandWorker
+
+sideEffect.then(() => {
+    worker = new OnDemandWorker(new URL('./maskwallet/worker.ts', import.meta.url), {
+        name: 'maskwallet',
+        type: 'module',
+    })
+    return () => worker.terminate()
```
Comment (guanbinrui) [inline packages/maskbook/src/plugins/Wallet/services/wallet/index.ts:12]: worker = null

File: cspell.json [hunk 5]
```diff
@@ -293,7 +293,9 @@
         "gamification",
         "Defi",
         "powah",
-        "Switcher"
+        "Switcher",
+        "maskwallet",
+        "acccount"
```
Comment (UncleBill) [inline cspell.json:298]:
```suggestion
        "maskwallet",
```

Don't need this.

File: packages/dashboard/src/pages/CreateMaskWallet/components/CreateMnemonic/index.tsx [hunk 6]
```diff
@@ -77,29 +77,20 @@ const CreateMnemonic = memo(() => {
 
     const [walletState, onSubmit] = useAsyncFn(async () => {
         const name = new URLSearchParams(location.search).get('name')
-        //#region if the name isn't exist, navigate to form page
+        // if the name isn't exist, navigate to form page
```
Comment (UncleBill) [inline packages/dashboard/src/pages/CreateMaskWallet/components/CreateMnemonic/index.tsx:80]:
```suggestion
        // if the name doesn't exist, navigate to form page
```

**Review by UncleBill — APPROVED** [review]

**Pull request merged**